src/containers/FactoryLoader.tsx

```diff
@@ -56,10 +59,12 @@ export class FactoryLoaderContainer extends React.PureComponent<Props, State> {
     super(props);
 
     const { search } = this.props.history.location;
+    const createPolicy = 'perclick';
```

Comment:
is it defaultCreatePolicy
```suggestion
    const defaultCreatePolicy = 'perclick';
```
?

Reply:
I have fixed it.